### DISCUSSION TITLE
With crazyswarm, Is it possible to get the LOG variables from firmware?

### DISCUSSION THREAD

**Discussion by IksuLoL** [USC-ACTLab/crazyswarm#722] [Q&A] [answered]
Hi,

We wrote some new modules in firmware and used the LOG and PARAM frameworks. To update the variables, which were registered using PARAM, works fine with crazyswarm. But I didn't find the API for getting the variables from the LOG section.

My question is, is it possible to read the logged variables with crazyswarm?

Thank you very much!

**Comment by knmcguire** [comment, marked as answer]
Check out the answer here https://github.com/USC-ACTLab/crazyswarm/discussions/612

**Reply by IksuLoL** [reply]
Thank you very much! I will try it!

**Comment by knmcguire** [comment]
Just mind, if you are not married with ROS1 and okay with switchin to ROS2, it would be a good idea to start using Crazyswarm2: https://imrclab.github.io/crazyswarm2/. 

There will be less support and maintance on Crazyswarm1 since the port so it will be more future proof.
